### PR TITLE
[MPH-197] Upgrade plexus-utils to 3.5.1

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/help/SystemMojo.java [hunk 1]
```diff
@@ -67,14 +67,10 @@ public void execute() throws MojoExecutionException {
...
         message.append(StringUtils.repeat("=", LINE_LENGTH)).append(LS);
-        try {
-            Properties envVars = CommandLineUtils.getSystemEnvVars();
-            for (String key : envVars.stringPropertyNames()) {
-                message.append(LS);
-                message.append(key).append("=").append(envVars.getProperty(key));
-            }
-        } catch (IOException e) {
-            getLog().warn("Unable to get the environment variables: " + e.getMessage());
+        Properties envVars = CommandLineUtils.getSystemEnvVars();
```
Comment (slawekjaranowski) [inline src/main/java/org/apache/maven/plugins/help/SystemMojo.java:70]: Why not `System.getenv()`

Comment (slachiewicz) [reply src/main/java/org/apache/maven/plugins/help/SystemMojo.java:70]: logic is more advanced https://github.com/codehaus-plexus/plexus-utils/blob/6714fe016acb5a8a503241c5f2b86438f387ac9f/src/main/java/org/codehaus/plexus/util/cli/CommandLineUtils.java#L374

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/help/SystemMojo.java:70]: It is only used for display - so maybe more complicated logic is not needed

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/help/SystemMojo.java:70]: Up to you

**Review by slawekjaranowski — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #324

**Comment by jira-importer** [comment]
Resolve #324

**Pull request merged**